tasks/build-test-addon.js

```diff
@@ -7,6 +7,7 @@ module.exports = function (grunt) {
   var glob = require('glob');
   var pkg = require('../package.json');
   var activeReporters = {};
+  grunt.option('firefox-bin', '/usr/bin/firefox');  // TODO more crossplatformy
```

Comment:
this means it won't work on mac right now?  maybe worth an equivalent of https://github.com/uProxy/node-chrome-runner